src/http/reply.cc

```diff
@@ -99,9 +108,8 @@ static const sstring& to_string(reply::status_type status) {
         return bad_gateway;
     case reply::status_type::service_unavailable:
         return service_unavailable;
-    default:
-        return internal_server_error;
     }
+    return internal_server_error;
```

Comment:
this is not strictly necessary or should be a separate commit. seastar devs will point this out.